CONTRIBUTING.md

```diff
@@ -150,14 +150,24 @@ See our separate [style guide](STYLE.md) document.
     to indicate their reach.
   - We publish detailed [release notes](https://www.cockroachlabs.com/docs/releases/)
     describing most non-test changes. To facilitate this, at least one commit in every
-    PR (possibly the PR message/merge commit) should contain a brief description of
+    PR (preferrably the PR message/merge commit) should contain a brief description of
     the change in terms that a user would recognize. This description should be prefixed
-    with "Release note:". For example, a commit like ["distsql: pre-reserve memory needed
+    with "Release note (category):", where the "category" is one of the following:
+    - cli change
+    - sql change
+    - ui change
+    - performance improvement
+    - bug fix
+    - general change (e.g., change of required go version)
+    - build change (e.g., compatibility with older CPUs)
+    - enterprise change (e.g., change to backup/restore)
+    - backwards-incompatible change
```

Comment:
Can you add a note about which category to choose when the commit falls into several categories? I assume you just pick the best match?

Reply:
Done.